app/assets/javascripts/application.js

```diff
@@ -19,3 +19,20 @@
 
 
 $(function(){ $(document).foundation(); });
+
+window.twttr = (function(d, s, id) {
+  var js, fjs = d.getElementsByTagName(s)[0],
+    t = window.twttr || {};
+  if (d.getElementById(id)) return t;
```

Comment:
That's fine, but factor this script out into its own file and call it here in the manifest rather than pasting it directly into the application.js file.